x-pack/platform/plugins/shared/stack_connectors/common/slack_api/schema.ts

```diff
@@ -68,10 +73,31 @@ export function validateBlockkit(text: string, ctx: z.RefinementCtx) {
   }
 }
 
+export function validateChannelName(value: string | undefined, ctx: z.RefinementCtx) {
+  if (!value || value.length === 0) {
+    ctx.addIssue({
+      code: z.ZodIssueCode.custom,
+      message: 'Channel name cannot be empty',
+    });
+    return;
+  }
+  if (!value.startsWith('#')) {
```

Comment:
What if the channel name is just `#`? Validation will pass, but the value is wrong. We specify `.min(1)`, but that might be wrong.